torch/utils/data/datapipes/dataframe/dataframes.py

```diff
@@ -0,0 +1,294 @@
+from typing import Any, Dict, List
+
+from torch.utils.data import (
+    DFIterDataPipe,
+    IterDataPipe,
+    functional_datapipe,
+)
+
+from torch.utils.data.datapipes.dataframe.structures import DataChunkDF
+
+# TODO(VitalyFedyunin): Add error when two different traces get combined
+class DataFrameTracedOps(DFIterDataPipe):
+    def __init__(self, source_datapipe, output_var):
+        self.source_datapipe = source_datapipe
+        self.output_var = output_var
+
+    def __iter__(self):
+        for item in self.source_datapipe:
+            yield self.output_var.calculate_me(item)
+
+
+#  TODO(VitalyFedyunin): Extract this list from the DFIterDataPipe registred functions
+DATAPIPES_OPS = ['dataframes_as_tuples', 'groupby', 'dataframes_filter', 'map', 'to_datapipe',
+                 'shuffle', 'concat', 'batch', 'dataframes_per_row', 'dataframes_concat', 'dataframes_shuffle']
```

Comment:
Are we exposing `dataframes_as_tuples`, `dataframes_per_row`, ... these API to users?

Reply:
I will underscore them =)